packages/store/ts/config/v2/defaults.ts

```diff
@@ -24,9 +24,9 @@ export const TABLE_DEPLOY_DEFAULTS = {
 export type TABLE_DEPLOY_DEFAULTS = typeof TABLE_DEPLOY_DEFAULTS;
 
 export const TABLE_DEFAULTS = {
-  namespace: "",
+  namespaceLabel: "",
   type: "table",
```

Comment:
just wonder why only these two properties are used as `TABLE_DEFAULTS`? Why is `namespace` not part of, and all the others in `TableInput`?

Reply:
I see now, `namespace` defaults to `namespaceLabel`.